rules/duo_rules/duo_admin_lockout.py

```diff
@@ -0,0 +1,12 @@
+def rule(event):
+    # Return True to match the log event and trigger an alert.
+    return event.get("action", "") == "admin_lockout"
+
+
+def title(event):
+    # If no 'dedup' function is defined, the return value
+    # of this method will act as deduplication string.
+    return (
+        f"Duo Admin [{event.get('username', '<NO_USER_FOUND>')}] is "
+        f"temporarily locked out due to too many passcode attempts."
```

Comment:
I'm not sure the description is static (aka `Admin temporarily locked out due to too many passcode attempts.` may only be 1 of many reasons'). Suggestion: 

pseudocode"

`f"locked out. Reason: event.get('description').get('message')"`